### PR TITLE
quiz rules page added

### PR DESCRIPTION
**Pull request by surbhi84**

### CODE REVIEW

**Review by aurobindoGupta — APPROVED** [review]
nice

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (Aditi0505) [inline .vscode/settings.json:1]: Can put this in gitignore file.

File: cssBundle/footer.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/* .copyright-text {
+  padding: 0 var(--spacing-xs);
+} */
```
Comment (Aditi0505) [inline cssBundle/footer.css]: Suggestion: Can remove commented code if not necessary

File: cssBundle/util.css [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+.text-dec-none {
+  text-decoration: none;
+}
+
+.mg-top-auto {
+  margin-top: auto;
+}
+
+.sec-ff {
+  font-family: "Ubuntu", sans-serif;
```
Comment (Aditi0505) [inline cssBundle/util.css]: Can put this in a variable as this is repeating.

File: cssBundle/variable.css [hunk 4]
```diff
@@ -0,0 +1,6 @@
+:root {
+  --prim-col: #cc5b7d;
+  --prim-col-li: #ff97b6c9;
+
+  --blk: black;
```
Comment (Aditi0505) [inline cssBundle/variable.css]: Can use a HEX value here to maintain consistency.

File: cssBundle/home.css [hunk 5]
```diff
@@ -0,0 +1,11 @@
+.cta-btn {
+  padding: var(--spacing-xs) var(--spacing-sm);
+  color: var(--prim-col);
+  font-family: "Ubuntu", sans-serif;
```
Comment (Aditi0505) [inline cssBundle/home.css]: Can put this in a variable since it is repeating.

**Review by aniketsingh13 — APPROVED** [review]
look good to me

File: index.html [hunk 6]
```diff
@@ -4,7 +4,121 @@
...
+    <main class="main-cont flex-center">
+      <div class="flex-col flex-center width80">
+        <h3 class="mg-sm">Welcome to checkBox,</h3>
+        <h3 class="marg-un">
+          A quiz app specifically designed for developers to have fun and learn
+          altogether.
+        </h3>
+
+        <h2>Are you ready?</h2>
+        <h1>Then lets begin!</h1>
```
Comment (Aditi0505) [inline index.html:63]: The hierarchy of heading is semantically incorrect here.  Is there a specific reason for writing this. let me know your thoughts about the same.

File: pages/quizCategories/quizCategories.css [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+.category-card > img {
+  width: 98%;
+  max-height: 60%;
+}
+
+.play-btn {
+  padding: var(--spacing-3xs) var(--spacing-2xs);
+  color: var(--prim-col);
+  font-family: "Ubuntu", sans-serif;
```
Comment (Aditi0505) [inline pages/quizCategories/quizCategories.css]: Can use a variable here.

**Review by Aditi0505 — APPROVED** [review]
Left few suggestions. Rest looks good to me.

File: cssBundle/footer.css (hunk 2 above)
Comment (surbhi84) [reply cssBundle/footer.css]: yeah removed it in the child branch

File: index.html (hunk 6 above)
Comment (surbhi84) [reply index.html:63]: i did that to grab attention where i intended to...i'll see if i can use font size.

Comment (Aditi0505) [reply index.html:63]: Yes, you can use font-size and font-weight property to achieve this.

**Pull request merged**